pkg/storage/chunk/aws/s3_storage_client_test.go

```diff
@@ -91,12 +92,14 @@ func Test_Hedging(t *testing.T) {
 		do            func(c *S3ObjectClient)
 	}{
 		{
-			"deletes are not hedged",
-			1,
+			"delete/put/list are not hedged",
+			3,
 			20 * time.Nanosecond,
 			10,
 			func(c *S3ObjectClient) {
 				_ = c.DeleteObject(context.Background(), "foo")
+				_, _, _ = c.List(context.Background(), "foo", "/")
```

Comment:
I don't understand this one, sorry; why would a `List` call (which appears to issue a `GET`?) not be hedged?

Reply:
The point is to hedge fetching chunks, which is the what is in the path of requests latency.

List is mostly used for listing rules, or index files, or the compactor.

Reply:
Gotcha :ok_hand: